civic_europe/assets/scss/components/_blocks.scss

```diff
@@ -213,7 +213,7 @@
             height: 3rem;
         }
 
-        img {
+        img { /* stylelint-disable-line */
```

Comment:
I am using this everywhere where we would need to re-position these lines because of the `no-descending-specifity`-rule. It is not trivial to change it so i am leaving it for now and just disable it

https://stylelint.io/user-guide/rules/list/no-descending-specificity

at some point we need to refactor this right @phillimorland ?

Reply:
@khamui it will depend if the site continues to be live